src/plugins/WebOfTrust/util/TickerDelayedBackgroundJobFactory.java

```diff
@@ -0,0 +1,70 @@
+package plugins.WebOfTrust.util;
+
+import freenet.support.Executor;
+import freenet.support.Ticker;
+
+/**
+ * Background job factory implementation for {@link TickerDelayedBackgroundJob}s.
+ *
+ * @author bertm
+ * @see TickerDelayedBackgroundJob
+ */
+public class TickerDelayedBackgroundJobFactory
+        extends BackgroundJobFactoryBase
+        implements DelayedBackgroundJobFactory {
+    /** The default delay. */
+    private final long defaultDelay;
+    /** The default ticker. */
+    private final Ticker ticker;
+
+    /**
+     * Constructs a background job factory with given default delay and {@link Ticker}.
+     * The {@link Ticker} given and its {@link Executor} <b>must</b> have an asynchronous
+     * implementation of respectively
+     * {@link Ticker#queueTimedJob(Runnable, String, long, boolean, boolean) queueTimedJob} and
+     * {@link Executor#execute(Runnable, String) execute}.
+     * @param delay the default trigger aggregation delay
+     * @param ticker an asynchronous ticker
+     */
+    public TickerDelayedBackgroundJobFactory(long delay, Ticker ticker) {
+        this.defaultDelay = delay;
+        this.ticker = ticker;
+    }
+
+    @Override
+    public TickerDelayedBackgroundJob newJob(Runnable job, String name) {
+        return newJob(job, name, defaultDelay, ticker);
+    }
+
+    /**
+     * Constructs a new {@link TickerDelayedBackgroundJob} using the default ticker. When
+     * this background job is {@link BackgroundJob#terminate() terminated}, the running job will
+     * be notified by interruption of its thread. Hence, the job implementer must take care not
+     * to swallow {@link InterruptedException}.
+     * @param job the job to run in the background
+     * @param name a human-readable name for the job
+     * @param delay the background job aggregation delay in milliseconds
+     * @see TickerDelayedBackgroundJob#TickerDelayedBackgroundJob(Runnable, String, long, Ticker)
+     */
+    @Override
+    public TickerDelayedBackgroundJob newJob(Runnable job, String name, long delay) {
+        return newJob(job, name, delay, ticker);
+    }
+
+    /**
+     * Constructs a new {@link TickerDelayedBackgroundJob}, overriding the default ticker. When this
+     * background job is {@link BackgroundJob#terminate() terminated}, the running job will be
+     * notified by interruption of its thread. Hence, the job implementer must take care not to
+     * swallow {@link InterruptedException}.
+     * @param job the job to run in the background
+     * @param name a human-readable name for the job
+     * @param delay the background job aggregation delay in milliseconds
+     * @param ticker the custom ticker
+     * @see TickerDelayedBackgroundJob#TickerDelayedBackgroundJob(Runnable, String, long, Ticker)
+     */
+    public TickerDelayedBackgroundJob newJob(Runnable job, String name, long delay, Ticker ticker) {
```

Comment:
This is also subject to the JavaDoc improment suggested here:
https://github.com/freenet/plugin-WebOfTrust/pull/26/files#r22342146

To not have to apply that change here as well, I would suggest to change the JavaDoc to:
```java
    /** Same as {@link #newJob(Runnable, String)} with the given delay and Ticker instead of the default. */
```

The linked function already inherits the JavaDoc from the parent, so the suggest improvement can be placed there.